admissions/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from common.util import get_semester_year_shorthand
+
+class Applicant(models.Model):
+    name = models.CharField(default="", null=False, max_length=100)
+    phone = models.CharField(default="", blank=True, max_length=50)
+    email = models.EmailField(unique=True)
+    date_of_birth = models.DateField(blank=True, null=True)
+    study = models.CharField(default="", blank=True, max_length=100)
+
+
+class Admissions(models.Model):
+    applicants = models.ForeignKey(Applicant)
```

Comment:
Shouldn't this be a `FK` from `Applicant` to `Admissions`, also needs an `on_delete` and `related_name` option